cmd/gardener-extension-admission-hcloud/app/app.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"context"
+	"fmt"
+
+	admissioncmd "github.com/23technologies/gardener-extension-provider-hcloud/pkg/admission/cmd"
+	hcloudinstall "github.com/23technologies/gardener-extension-provider-hcloud/pkg/hcloud/apis/install"
+	providerhcloud "github.com/23technologies/gardener-extension-provider-hcloud/pkg/hcloud"
+
+	controllercmd "github.com/gardener/gardener/extensions/pkg/controller/cmd"
+	"github.com/gardener/gardener/extensions/pkg/util"
+	webhookcmd "github.com/gardener/gardener/extensions/pkg/webhook/cmd"
+	"github.com/gardener/gardener/pkg/apis/core/install"
+	"github.com/spf13/cobra"
+	componentbaseconfig "k8s.io/component-base/config"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+)
+
+var log = logf.Log.WithName("gardener-extension-admission-aws")
+
+// NewAdmissionCommand creates a new command for running an AWS admission webhook.
```

Comment:
Could you move this stuff to `pkg/cmd/admission` by chance? You may want to take `pkg/cmd/controller` as reference. This makes the "cmd" directory containing main entrypoints only and moves the remaining code to "pkg" where it belongs to IMHO.